io-functions-pay-portal/utils/config.ts

```diff
@@ -7,17 +7,20 @@
 
 import * as t from "io-ts";
 import { readableReport } from "italia-ts-commons/lib/reporters";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
 export const IConfig = t.interface({
-  isProduction: t.boolean
+  IO_PAGOPA_PROXY_PROD_BASE_URL : NonEmptyString,
+  IO_PAGOPA_PROXY_TEST_BASE_URL : NonEmptyString,
```

Comment:
`Healthcheck.ts` checks that `IO_PAGOPA_PROXY_TEST_BASE_URL` is set. Is it correct to assume that it is required? or is it better to be optional?

Reply:
Yes it should be required, we want to be able to test the payments